### DISCUSSION TITLE
Is here a fromNodeMiddleware but for error hanfler ((err, req,res, next) => {})

### DISCUSSION THREAD

**Discussion by Maxttier** [nitrojs/nitro#2951] [Help Wanted]
### Environment

latest

### Reproduction

fromNodeMiddleware((error, req, res, next) => {})

### Describe the bug

I think that not work h3App.use(fromNodeMiddleware((error, req, res, next) => {})) or  h3App.router.use(fromNodeMiddleware((error, req, res, next) => {}))

```
const bodyParser = require('body-parser')
const methodOverride = require('method-override')

app.use(bodyParser.urlencoded({
  extended: true
}))
app.use(bodyParser.json())
app.use(methodOverride())
app.use((err, req, res, next) => {
  // logic
})
```

**Comment by Maxttier** [comment]
defineNitroErrorHandler

**Comment by pi0** [comment]
Nitro uses h3 as runtime server which supports global error handlers instead of middleware style of express.

`fromNodeMiddleware` is a compatibility layer to plug an existing legacy connect/express based app to nitro but only for compatibility.

As you noticed, you should use `errorHandler` config + an entry that defines the handler using `defineNitroErrorHandler`

Example: https://stackblitz.com/edit/github-weasgzp4-yfodvndx?file=nitro.config.ts,server%2Ferror.ts
